src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -808,6 +811,33 @@ private void paste() {
         }
     }
 
+    private void copyTitle() {
+        List<BibEntry> bes = mainTable.getSelectedEntries();
+        if (!bes.isEmpty()) {
+            storeCurrentEdit();
+            List<String> titles = new ArrayList<>(bes.size());
+
+            // Collect all non-null keys.
+            for (BibEntry be : bes) {
+                be.getTitle().ifPresent(titles::add);
```

Comment:
It also works with flatMap and a helper in OptionalUtil (not sure how the method is called that transforms an optional into a stream)